tests/detectors/test_queued_detector.py

```diff
@@ -12,7 +12,7 @@
 from unittest import mock
 
 from preggy import expect
```

Comment:
```suggestion
import pytest
from preggy import expect
```

Reply:
I've changed all the duplicate code inside `setUp`, so no need to import `pytest` anymore